docs/source/sql/ml.rst

```diff
@@ -85,7 +78,7 @@ Example:
 .. raw:: html
 
     <div class="highlight-sql notranslate"><div class="highlight"><pre><span></span><span class="k">CREATE MODEL</span> <span class="n">my_model</span> <span class="k">WITH</span> <span class="p">(</span>
-        <span class="n">model_class</span> <span class="o">=</span> <span class="s1">'dask_ml.xgboost.XGBClassifier'</span><span class="p">,</span>
```

Comment:
Why not point to `xgb.dask.DaskXGBClassifier` or `xgboost.XGBClassifier` ?


See docs: 
https://xgboost.readthedocs.io/en/stable/python/python_api.html#xgboost.XGBClassifier

Reply:
Makes sense, I'll update that now.